openpype/settings/defaults/project_settings/maya.json

```diff
@@ -169,6 +169,55 @@
             "enabled": false,
             "attributes": {}
         },
+        "ValidateLoadedPlugin": {
+            "enabled": false,
+            "authorized_plugins": [
```

Comment:
@davidlatwe had a very good idea: As for the ‘default’ list, wonder if we could just filter out loaded plugins that are under MAYA_LOCATION dynamically in that validation plugin. 

That would allow keeping this list clean for any additional plugins the studio would like to allow.

Reply:
Very Good idea, I am looking at how to do this.